app/src/main/java/com/sebastianvm/mango/ui/example/ExampleScreen.kt

```diff
@@ -56,7 +60,7 @@ fun ExampleScreenLayout(
             Text(text = "Add income source")
         }
 
-        LazyColumn {
+        LazyColumn(contentPadding = WindowInsets.navigationBars.asPaddingValues()) {
```

Comment:
Note: `contentPadding` can be used to add padding before the first or after the last item. Using the `modifier.padding` instead would result on it being added to the list (e.g. items would not show up behind the padding)